hedera-node/configuration/compose/log4j2.xml

```diff
@@ -211,6 +222,10 @@
       <AppenderRef ref="Console"/>
       <AppenderRef ref="RollingFile"/>
     </Logger>
+    <Logger name="com.hedera.node.app.blocks.impl.streaming" level="INFO" additivity="false">
+      <AppenderRef ref="Console"/>
```

Comment:
Should we really be sending this to the standard output channel at an `INFO` level?